### PR TITLE
Fix bloom-filter

### PR DESCRIPTION
**Pull request by inwonkim**
1. Fix bloom filter.
2. Modify bloom-filter related tests

### CODE REVIEW

File: iconservice/utils/bloom.py [hunk 1]
```diff
@@ -19,12 +19,12 @@ def get_chunks_for_bloom(value_hash):
 
 
 def chunk_to_bloom_bits(chunk):
-    high, low = bytearray(chunk.encode())
+    high, low = chunk
```
Comment (eunsoo-icon) [inline iconservice/utils/bloom.py]: Recommend keeping original code.
```suggestion
    high, low = bytearray(chunk)
```

Comment (eunsoo-icon) [reply iconservice/utils/bloom.py]: Remove changes comment about chunk_to_bloom_bits() at the beginning of file

Comment (inwonkim) [reply iconservice/utils/bloom.py]: Removed changes comment about chunk_to_bloom_bits() at the beginning of the file.

Comment (inwonkim) [reply iconservice/utils/bloom.py]: Reverted to original bloom-filter code

**Pull request merged**